modules/msp/apm/trace/core/query/query.go

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) 2021 Terminus, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package query
+
+import (
+	"encoding/json"
+	"strings"
+
+	"github.com/erda-project/erda-infra/providers/i18n"
+	"github.com/erda-project/erda-proto-go/msp/apm/trace/pb"
+)
+
+type SpanTree map[string]*pb.Span
+type ConditionType string
+
+const (
+	INPUT ConditionType = "input"
+)
+
+var sortConditions = []*pb.TraceQueryCondition{
+	{Key: strings.ToLower(pb.SortCondition_TRACE_TIME_DESC.String()), Value: strings.ToLower(pb.SortCondition_TRACE_TIME_DESC.String())},
```

Comment:
Use config.yml instead of hard coding

Reply:
After adding the global configuration reader, optimize the code